src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/memory.cpp

```diff
@@ -6,10 +6,62 @@
 
 #include <vector>
 
+#include "ngraph/opsets/opset11.hpp"
```

Comment:
Please use the new namespaces (ov::, openvino):

#include "openvino/opsets/opset11.hpp"
and
using namespace ov::opset11;